client/src/pages/MainMenu.js

```diff
@@ -3,19 +3,63 @@ import styled from "styled-components";
 
 import GlobalStyle from "../GlobalStyles";
 import { HeaderMenu } from "../components/HeaderMenu";
+import { Button } from "../components/Button";
+import { useHistory } from "react-router-dom";
 
 const PageWrapper = styled.div`
-  height: 100%;
+  display: flex;
+  height: 100vh;
   width: 100%;
   background: var(--gradient-dark);
 `;
 
+const MenuWrapper = styled.div`
+  align-self: flex-end;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 1.5rem 2rem;
+  width: 100%;
+  max-width: 400px;
+  margin: 0 auto;
+  Button {
```

Comment:
I don't think this rule works. Maybe you wanted to do this?

```suggestion
  ${Button} {
```

Reply:
Thanks. Adopted this change into the code 😄

Reply:
Tried both options. Original version ends up with a working result, suggestion ends up breaking the code.

Reply:
Weird :) 
It should be possible with https://styled-components.com/docs/advanced#referring-to-other-components

`Button` is not referring to the component, it's referring to the `button` element I think. 
This doesn't seems correct. Maybe take a look at this issue another day